scripts/apps/search-providers/directive.ts

```diff
@@ -41,6 +41,12 @@ export default function SearchProviderConfigDirective(searchProviderService, not
              * Upserts the selected search provider.
              */
             $scope.save = function() {
+                // check for config
+
+                if ($scope.origProvider?.config && $scope.provider?.config) {
```

Comment:
What about other fields? If I added `config2` property, would the same issue occur? It would be good to write code that would solve the issue no matter the field name.